.circleci/config.yml

```diff
@@ -57,3 +57,4 @@ workflows:
   build:
     jobs:
       - "test"
+    when: false  # disable this workflow until Percy tests are functional again
```

Comment:
Taken from https://til.codeinthehole.com/posts/how-to-easily-disable-a-circleci-workflow/

Pretty neat!